### DISCUSSION TITLE
Smaller font sizes

### DISCUSSION THREAD

**Discussion by ohmmee** [community/community#77726] [Accessibility]
the minimum font override size is still too big, there still needs to be 5 more steps that reduces font size in "Override System Fontsize" option in code options in settings.

**Comment by ghostinhershell** [comment]
Hey there @ohmmee, 

Thanks so much for sharing your feedback here in the community. Before sharing this with the Accessibility team, can you add more details about the issue that you're running into here? For example, how is the current font size limit impacting you? Any additional details would be helpful!

**Comment by ohmmee** [comment]
Hi Sarto Jama,

Thanks for getting back to the feedback. I've attached a few screenshots of
my iphone and ipad.
I don't code on these devices but use the app to check the code.

As you can see from the attachments, only a section of the code is in the
viewport.
I'm aware of the line wrapping feature but it is disorienting to look at
without structure.

Had it been that, there was an option to further decrease the font size in
slider values,
the code would've been visible on the single long trace as in VSCode.
Since, these phones already have higher text resolution, smaller texts
would've also read fine.

Thanks. Regards

**Comment by cehfisher** [comment]
Thank you for your feedback @ohmmee - we genuinely appreciate your input. Our team has reviewed your request and we want to be transparent about our priorities. This particular request is not currently being actively worked on. This doesn’t imply that we’re not considering it in the future, but for now, it is not on our development roadmap.

We encourage you to check our [public roadmap](https://github.com/github/roadmap/projects/1) to get insights into the projects we are currently focusing on for the upcoming year. Please note that smaller fixes may not always be listed explicitly but can often be addressed when related work is in progress.

As a valuable GitHub user, your feedback in this Discussion space, accompanied by appropriate tags and examples, plays a crucial role in shaping GitHub’s development. While we can’t guarantee immediate action on every piece of feedback, rest assured that we are attentive to your suggestions and always striving to enhance the GitHub experience.

Once again, thank you for taking the time to share your thoughts with us, even if the particular request is not currently in active development. Your input is valuable and contributes to the continuous improvement of GitHub!
